### ISSUE TITLE
write typical react component and integrate with hyperstack

### ISSUE THREAD

**Issue by Jothinayagan** [hyperstack-org/hyperstack#439]
There is a requirement. I wanted to code a typical react component in js and integrate it with hyperstack.

Is it really possible? I have tried but nothing works. 

Any idea or suggestion?

**Comment by mpantel** [comment]
Yes, you can see how in the main documentation: https://docs.hyperstack.org/client-dsl/javascript-components#importing-javascript-or-react-libraries
Hyperstack depends on React 16 though...

**Comment by Jothinayagan** [comment]
Thanks for the response @mpantel . The docs that you shared shows how to use react libs. I wanted to use custom react component.

for example,

```javascript
function Car() {
  return <h2>Hi, I am a Car!</h2>;
}

export default Car;
```

I wanted to execute this file as it is. Is there anyway to do this?

**Comment by Jothinayagan** [comment]
@mpantel  @catmando Any suggestions to integrate pure react component in hyperstack?
